src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -62,11 +106,20 @@ def _create_session(self, alias, url, headers, cookies, auth,
 
         `verify` set to True if Requests should verify the certificate
         
-        `debug` enable http verbosity option more information https://docs.python.org/2/library/httplib.html#httplib.HTTPConnection.set_debuglevel        
+        `debug` enable http verbosity option more information 
+                https://docs.python.org/2/library/httplib.html#httplib.HTTPConnection.set_debuglevel
         
         `max_retries` The maximum number of retries each connection should attempt.
+        
+        `max_delay` The maximum number of delay each connection should attempt.
         """
-
+        
+        # Bringing global variable and type casting since robot vars unicode
+        global MAX_RETRIES, MAX_DELAY, MAX_BACKOFF
```

Comment:
I thought i could use global variables not an good idea :-1: 

I have modified code, if it is OK :+1:  i will submit 

``` Python
def retry(ExceptionToCheck, logger=None):
    """
    Retry calling the decorated function using an exponential backoff.
    """
    def deco_retry(f):
        def f_retry(self, *args, **kwargs):                       
                mretries, mdelay, mbackoff = self.mretries, self.mdelay, self.mbackoff           
```

``` Python
def __init__(self):
        self._cache = robot.utils.ConnectionCache('No sessions created')
        self.builtin = BuiltIn()        
        self.debug = 0
        self.mretries = 0
        self.mdelay = 0.0
        self.mbackoff = 0.0
```
``` Python
def _create_session(self, alias, url, headers, cookies, auth,
                        timeout, proxies, verify, debug, max_retries, max_delay, max_backoff):
      # Type casting since robot vars unicode
        self.mretries = int(max_retries)
        self.mdelay = float(max_delay)
        self.mbackoff = float(max_backoff)
```

``` Python
 @retry(Exception, logger)
    def get_request(self, alias, uri, headers=None, params={}, allow_redirects=None, timeout=None):
```

Any comments/suggestions?